src/main/java/com/blackhearth/blockchain/SimpleBlockchainApplication.java

```diff
@@ -10,8 +13,14 @@
 @SpringBootApplication
 public class SimpleBlockchainApplication {
 
+	@SneakyThrows
 	public static void main(String[] args) {
 		SpringApplication.run(SimpleBlockchainApplication.class, args);
 		Map<RunParams, Object> params = ParamsReader.getParams(args);
+
+		BasicPeerToPeerService basicPeerToPeerService = new BasicPeerToPeerService(new InterpreterBlockChainCommunication());
+		basicPeerToPeerService.start(1234);
```

Comment:
port losowy. dobrze aby była informacja na jakim porcie węzeł powstał.

Reply:
resolved, ten kod jest tutaj bugiem.

Reply:
To nie miał być docelowy kod, tylko taki 'pokaz' jak z tego można skorzystać